entity-management-web/src/main/java/eu/europeana/entitymanagement/web/service/EntityRecordService.java

```diff
@@ -229,7 +229,7 @@ public EntityRecord createEntityFromRequest(EntityPreview entityCreationRequest,
 		 * sameAs will be replaced during consolidation; however we set this here to prevent duplicate
 		 * registrations if consolidation fails
 		 */
-		entity.setSameAs(Collections.singletonList(entityCreationRequest.getId()));
+		entity.setSameReferenceLinks(Collections.singletonList(entityCreationRequest.getId()));
```

Comment:
the sameAs is available in the request as well, probably it is better to merge that values as well

Reply:
I don't understand this remark.